descholar-ceo/AnnounceIT
# #170726268 User create announcement

#### What does this PR ​ do​ ?
User should be able to create new announcement

#### Description of Task to be completed?
* User clicks on the create button from his/her homepage,
* He/she get redirected to the create new announcement page
* After successful announcement creation, user get redirect to his/her profile page

 #### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

*Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/user-profile.html`, will be redirected to the profile page and click on create button in navbar, you will be immediatelly got to the create new announcement page, or enter like this `http://127.0.0.1:5500/UI/html/create-announcement.html`

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `create-announcement.html` and right click and open with your preferred browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170726268](https://www.pivotaltracker.com/n/projects/2429498)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-18 23-52-50](https://user-images.githubusercontent.com/56886064/72671079-e2c9f300-3a4d-11ea-8f46-81855c659e71.png)
![Screenshot from 2020-01-18 23-53-05](https://user-images.githubusercontent.com/56886064/72671081-e2c9f300-3a4d-11ea-9850-065b1337b05b.png)


#### Questions:
No Question
